tests/functional/storageclass/test_csi_subvolume_group_property.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import yaml
+
+from ocs_ci.framework.testlib import ManageTest, tier3, skipif_external_mode
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.helpers.performance_lib import run_oc_command
+
+logger = logging.getLogger(__name__)
+
+
+@green_squad
+@skipif_external_mode
+@tier3
+class TestCSISubvolumeGroup(ManageTest):
+    def test_network_fence_not_editable(self):
```

Comment:
The test name doesn't seem to match with the flow of the test. any reason as to why the name is `test_network_fence_not_editable`?

Reply:
Changed to a meaningful name , of course that was a typo that should have been fixed.